Tweak default icewm preferences

Attention: this patch is applied in the %build phase, because it patches
a file generated by ./src/genpref. To edit the patch:

  quilt setup icewm.spec
  cd icewm-<version>
  quilt push -a
  ./configure <options from the spec>
  make -j8 # Generate src/preference
  quilt import -p0 patches/icewm-preferences.patch
  quilt push
  vi lib/preferences
  quilt refresh
---
Index: icewm-3.3.0/src/preferences
===================================================================
--- icewm-3.3.0.orig/src/preferences
+++ icewm-3.3.0/src/preferences
@@ -207,6 +207,7 @@
 
 #  Icon search path (colon separated)
 # IconPath="/usr/local/share/icons:/usr/local/share/pixmaps:/usr/share/icons:/usr/share/pixmaps:"
+IconPath="/usr/share/icons/Adwaita/16x16/legacy:/usr/share/icons/Adwaita/16x16/apps:/usr/share/icons/hicolor:/usr/share/icons:/usr/share/pixmaps"
 
 #  Colon separated icon theme list with wildcard support. Minus prefix - can be used to exclude themes.
 # IconThemes="*:-HighContrast"
@@ -501,6 +502,7 @@
 
 #  Command to lock display/screensaver.
 # LockCommand=""
+LockCommand="xlock -mousemotion"
 
 #  Command to cancel logout.
 # LogoutCancelCommand=""
@@ -702,6 +704,7 @@
 
 #  Command to reboot the system.
 # RebootCommand="test -e /run/systemd/system && systemctl reboot"
+RebootCommand="/bin/sh -c 'systemctl reboot ||:'"
 
 #  Send the clicks outside menus to target window.
 # ReplayMenuCancelClick=0 # 0/1
@@ -771,6 +774,7 @@
 
 #  Command to shutdown the system.
 # ShutdownCommand="test -e /run/systemd/system && systemctl poweroff"
+ShutdownCommand="/bin/sh -c 'systemctl poweroff ||:'"
 
 #  Maximized windows can be resized.
 # SizeMaximized=0 # 0/1
@@ -1473,18 +1477,21 @@ WorkspaceNames=" 1 ", " 2 ", " 3 ", " 4
 
 #  Display desktop background centered and not tiled.
 # DesktopBackgroundCenter=0 # 0/1
+DesktopBackgroundCenter=0 # 0/1
 
 #  Desktop background color(s)
 # DesktopBackgroundColor=""
 
 #  Desktop background image(s)
 # DesktopBackgroundImage=""
+DesktopBackgroundImage="BRANDING_PICTURE"
 
 #  Paint the background image over all multihead monitors combined.
 # DesktopBackgroundMultihead=0 # 0/1
 
 #  Resize desktop background to full screen.
 # DesktopBackgroundScaled=0 # 0/1
+DesktopBackgroundScaled=1 # 0/1
 
 #  Color(s) to announce for semitransparent windows.
 # DesktopTransparencyColor=""
